static/js/containers/FinancialAidCalculator.js

```diff
@@ -56,7 +55,7 @@ const currencySelect = (update, current) => (
 
 const salaryUpdate = R.curry((update, current, e) => {
   let newEdit = R.clone(current);
-  newEdit.income = sanitizeNumberString(20, e.target.value);
+  newEdit.income = e.target.value
```

Comment:
just undo the change you made to line 59 in this file, keep it as-is on master. `sanitizeNumberString` basically just removes leading zeros.